src/test/java/io/dashboardhub/pipelinedashboard/definitions/ProjectDefinitions.java

```diff
@@ -10,6 +10,8 @@
 import org.junit.Assert;
 import org.springframework.beans.factory.annotation.Autowired;
 
+import java.util.UUID;
```

Comment:
This is a leftover from when I was using proper UUIDs. It'll be needed when we use them, but otherwise it can just be removed.